depends/common/libraw/CMakeLists.txt

```diff
@@ -7,7 +7,12 @@ include(ExternalProject)
 externalproject_add(libraw
   SOURCE_DIR ${CMAKE_SOURCE_DIR}
   UPDATE_COMMAND autoreconf -vif
-  CONFIGURE_COMMAND CC=${CMAKE_C_COMPILER} AR=${CMAKE_AR} ${CMAKE_SOURCE_DIR}/configure --disable-examples --prefix=${OUTPUT_DIR}
+  CONFIGURE_COMMAND CC=${CMAKE_C_COMPILER} AR=${CMAKE_AR}
+                    ${CMAKE_SOURCE_DIR}/configure
+                      --disable-examples
+                      --disable-shared
+                      --with-pic
+                      --prefix=${OUTPUT_DIR}
```

Comment:
Also by this without version change? This is the major fix.

Reply:
I'll handle creating a new release after this PR is merged.